.github/workflows/chromatic.yml

```
@@ -0,0 +1,43 @@
+
+      - name: Install dependencies
+        run: npm --prefix pho-chat ci
+
+      - name: Generate tokens MDX
+        run: npm --prefix pho-chat run tokens:mdx
+
+      - name: Build Storybook
+        run: npm --prefix pho-chat run storybook:build
+
+      - name: Publish to Chromatic
+        if: ${{ secrets.CHROMATIC_PROJECT_TOKEN != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)